core/primitives/version.hpp

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_PRIMITIVES_VERSION_HPP_
+#define KAGOME_CORE_PRIMITIVES_VERSION_HPP_
+
+#include <array>
+#include <cstdint>
+#include <vector>
+
+namespace kagome::primitives {
+
+  /// The identity of a particular API interface that the runtime might provide.
+  using ApiId = std::array<uint8_t, 8>;
+
+  using Api = std::pair<ApiId, uint32_t>;
+
+  /// A vector of pairs of `ApiId` and a `u32` for version. For `"std"` builds,
+  /// this
```

Comment:
```suggestion
  /// this is a 'Cow'
```
Also, what does it mean?